alephbet.js.coffee

```diff
@@ -92,6 +92,9 @@ class AlephBet
     add_goal: (goal) =>
       goal.add_experiment(this)
 
+    add_goals: (goals) =>
+      goal.add_experiments(this)
```

Comment:
this will fail actually... you're passing `goals`, but using `goal` here... I would simply mirror what you did on line 116.

or you can also use a more coffeescript-friendly way:

```coffeescript
add_goals: (goals) =>
  @add_goal(goal) for goal in goals
```